build.sbt

```diff
@@ -1,5 +1,5 @@
 name := "scala-js-d3v4"
-version := "master-SNAPSHOT"
+organization := "com.github.fdietze.scala-js-d3v4"
```

Comment:
Will probably revert this. Didn't intend to commit it, but it's here to help with `publishLocal` for testing in another project.

Reply:
If it does not confict with jitpack publishing, I'm fine with leaving it in the code.

Reply:
@fdietze, I'll try testing with an account of my own and get back to you.